.github/workflows/push.yml

```diff
@@ -14,8 +14,8 @@ jobs:
         GH_ADMIN_TOKEN: ${{ secrets.GH_ADMIN_TOKEN }}
         PR_ACTIVE_BRANCH: master
         PR_BRANCH_NAME: automated-java-snippets-update
-        PR_SUBMODULE_PATH: .repos/nexmo/nexmo-java-code-snippets
+        PR_SUBMODULE_PATH: .repos/vonage/vonage-java-code-snippets
         PR_TARGET_BRANCH: master
         PR_TARGET_ORG: nexmo
```

Comment:
should be vonage.

Reply:
nexmo-developer repo is still located under the nexmo org

Reply:
@dragonmantank - thoughts on this? should this not be updated until after the submodules are updated or do you think we're good to push it now?

Reply:
@slorello89 I spoke with @dragonmantank and he confirmed this file should be updated so I changed it back.